src/applications/gi/components/vet-tec/VetTecCalculator.jsx

```diff
@@ -0,0 +1,186 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import { isEmpty } from 'lodash';
+
+import LoadingIndicator from '@department-of-veterans-affairs/formation-react/LoadingIndicator';
+import {
+  calculatorInputChange,
+  beneficiaryZIPCodeChanged,
+} from '../../actions';
+import { getCalculatedBenefits } from '../../selectors/vetTecCalculator';
+import VetTecCalculatorForm from './VetTecCalculatorForm';
+
+export class VetTecCalculator extends React.Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      tuitionFees: 0,
+      scholarships: 0,
+    };
+  }
+
+  renderCalculatorForm = () => {
+    const {
+      calculator: inputs,
+      calculated: { inputs: displayed },
+    } = this.props;
+
+    return (
+      <div className="calculator-inputs">
+        <div className="form-expanding-group-open">
+          <VetTecCalculatorForm
+            inputs={inputs}
+            displayedInputs={displayed}
+            onInputChange={this.props.calculatorInputChange}
+            onBeneficiaryZIPCodeChanged={this.props.beneficiaryZIPCodeChanged}
+          />
+        </div>
+      </div>
+    );
+  };
+
+  renderTuitionSection = outputs => (
+    <div className="tuition-section">
+      <div className="row calculator-result">
+        <div className="small-6 columns">
+          <h5>Tuition & fees:</h5>
+        </div>
+        <div className="small-6 columns value">
+          <h5>{outputs.vetTecTuitionFees}</h5>
+        </div>
+      </div>
+
+      <div className="row calculator-result">
+        <div className="small-6 columns">
+          <div>Your scholarships:</div>
+        </div>
+        <div className="small-6 columns value">
+          <div>{outputs.vetTecScholarships}</div>
+        </div>
+      </div>
+
+      <div className="row calculator-result">
+        <div className="small-8 columns">
+          <div>
+            VA pays to provider:{' '}
+            <a href="" target="_blank" rel="noopener noreferrer">
+              (Learn more)
+            </a>
+          </div>
+        </div>
+        <div className="small-4 columns value">
+          <div>{outputs.vaPaysToProvider}</div>
+        </div>
+      </div>
+
+      <div className="row calculator-result va-payment">
```

Comment:
Can use a combination of the [responsive prefixes](https://design.va.gov/utilities/padding#responsive-prefixes), [padding-left](https://design.va.gov/utilities/padding#padding-left), and [padding-right](https://design.va.gov/utilities/padding#padding-right) utilities instead of `va-payment` with a media query. So something like: `medium-screen:vads-u-padding-left--1 medium-screen:vads-u-padding-right--7`

Reply:
Thanks!